eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -1235,6 +1235,18 @@ case class Commitments(params: ChannelParams,
   def resolveCommitment(spendingTx: Transaction): Option[Commitment] = {
     all.find(c => spendingTx.txIn.map(_.outPoint).contains(c.commitInput.outPoint))
   }
+
+  /**
+   * Returns the set of splice transactions that spend the given commitment.
+   */
+  def resolveSpendingFundingTxs(commitment: Commitment): Set[ByteVector32] = {
```

Comment:
I can't find a clearer name for that function, but I'm not very satisfied with it...if you have better ideas let me know! Also I put this in `Commitments`, but it could also have been a static function in `CommonFundingHandlers`, I'm not sure what's best.